src/MLJGLMInterface.jl

```diff
@@ -122,16 +125,32 @@ function prepare_inputs(model, X; handle_intercept=false)
 end
 
 """
-    glm_report(fitresult)
+    glm_report(glm_model, features)
 
-Report based on the `fitresult` of a GLM model.
+Report based on the fitted `LinearModel/GeneralizedLinearModel`, `glm_model`.
 """
-function glm_report(fitresult)
-    deviance = GLM.deviance(fitresult)
-    dof_residual = GLM.dof_residual(fitresult)
-    stderror = GLM.stderror(fitresult)
-    vcov = GLM.vcov(fitresult)
-    return (; deviance=deviance, dof_residual=dof_residual, stderror=stderror, vcov=vcov)
+function glm_report(glm_model, features)
+    deviance = GLM.deviance(glm_model)
+    dof_residual = GLM.dof_residual(glm_model)
+    stderror = GLM.stderror(glm_model)
+    vcov = GLM.vcov(glm_model)
+    coef_table = GLM.coeftable(glm_model)
+    # Update the variable names in the `coef_table` with the actual variable
+    # names seen during fit.
+    if length(coef_table.rownms) == length(features)
+        # This means `fit_intercept` is false
+        coef_table.rownms = string.(features)
+    else      
+        coef_table.rownms = [
+            (string(features[i]) for i in eachindex(features))...; "(Intercept)"
+        ]
```

Comment:
Here a string and in the final API a symbol? I see that a symbol is returned in the test at least. Is that correct?

Reply:
Yeah, a `StatsBase.CoefTable` object only accepts string feature names.